pkg/cmd/log.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"github.com/puppetlabs/nebula-sdk/pkg/log"
+	"github.com/spf13/cobra"
+)
+
+func NewLogCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:                   "log",
+		Short:                 "Submit annotated log messages",
+		DisableFlagsInUseLine: true,
+	}
+
+	cmd.AddCommand(NewLogInfoCommand())
+	cmd.AddCommand(NewLogWarnCommand())
+	cmd.AddCommand(NewLogErrorCommand())
+	cmd.AddCommand(NewLogFatalCommand())
+
+	return cmd
+}
+
+func NewLogInfoCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   string(log.LogLevelInfo),
+		Short: "Logs an informational message",
+		Run: func(cmd *cobra.Command, args []string) {
+			log.Info(args[0])
```

Comment:
A few very minor things here:
* You might want to specify `Args: cobra.MinimumNArgs(1)` ([doc](https://github.com/spf13/cobra#positional-and-custom-arguments)) on these commands to prevent a panic if no args are specified.
* For commands like this (and e.g. `echo` etc.) people tend to want to be able to leave off the quotes for simple log messages, e.g. `echo foo bar` and `echo "foo bar"` should behave basically the same way (save for some whitespace cleanup that the shell does). You might want to make these something like `log.Info(strings.Join(args, " "))` so people can do that.